src/main.js

```diff
@@ -30,8 +30,8 @@ var items = [
   }
 ];
 
-main();
+// Expose globally for debugging purpose
+window.React = React;
```

Comment:
Questo è importante per il debugging, prova a lanciare la build di prod, e usa chrome in debug, vedrai